packages/components/src/Markdown/Markdown.tsx

```diff
@@ -26,22 +32,37 @@ interface BaseRendererProps {
   children: ReactNode;
 }
 
-export function Markdown({ content, externalLink }: MarkdownProps) {
+export function Markdown({ content, externalLink, basicUsage }: MarkdownProps) {
+  const props = {
+    ...(basicUsage && {
+      disallowedTypes: ["paragraph", "heading"],
```

Comment:
Should we allow `images` and `tables`, etc here?

https://www.markdownguide.org/cheat-sheet

Reply:
nope!